### ISSUE TITLE
fix(fab): reset tab indices

### ISSUE THREAD

**Issue by EladBezalel** [angular/material#8566]
On desktop, we should be able to reset the indexes so you cannot tab through, but this breaks accessibility, especially on mobile, since you have no arrow keys to press

> Written originally by @topherfangio [here](https://github.com/angular/material/blob/master/src/components/fabSpeedDial/fabController.js#L169-L170)

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our ‘Surge Focus on Material 2' efforts.
        For details, see our [forum posting](http://bit.ly/1UhZyWs).

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our ‘Surge Focus on Material 2' efforts.
        For details, see our [forum posting](http://bit.ly/1UhZyWs).

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our ‘Surge Focus on Material 2' efforts.
        For details, see our [forum posting](http://bit.ly/1UhZyWs).

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our ‘Surge Focus on Material 2' efforts.
      For details, see our forum posting @ http://bit.ly/1UhZyWs.

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our ‘Surge Focus on Material 2' efforts.
      For details, see our forum posting @ http://bit.ly/1UhZyWs.
